src/components/PreviewFrame.jsx

```diff
@@ -122,9 +139,14 @@ class PreviewFrame extends React.Component {
 }
 
 PreviewFrame.propTypes = {
+  highlighterSelector: PropTypes.string,
   src: PropTypes.string.isRequired,
   onRuntimeError: PropTypes.func.isRequired,
 };
 
+PreviewFrame.defaultProps = {
+  highlighterSelector: '',
```

Comment:
I would probably make the default `null` here.

Reply:
Changed to null